documentation/for_developers.rst

```diff
@@ -73,10 +73,16 @@ The tags created by the user are stored here for easy access. The tags can be so
 The Images section shows all the images taken and they can come in in real time. The images are seperated into three main groups: images that have been reviewed, images that have not been reviewed, and all images. 
 
 **Map**
-If an image is selected for review, it is put onto the Map where the user can edit tags in the image
+If an image is selected for review, it is displayed onto the right portion of the window where the user can add or remove tags on the image.
 
 What controls there are
 ~~~~~~~~~~~~~~~~~~~~~~~
 
 **Adding Tags**
 To add tag an object in the image, first click add under the Tags section and fill in corresponding characteristics. Then hover over object in the image and right click to tag it. 
+
+**Determining Geolocation**
```

Comment:
Great thing to mention. It is a property of the Main Window however -- speaking of which, we should look into making a section for it this Saturday. For now, I suggest you save this section on your laptop, remove it from this commit, and we add it where it belongs on Saturday.

Reply:
It has been removed